### PR TITLE
[bn] localize content/bn/includes/partner-style.css for Bengali localization

### PR DESCRIPTION
**Pull request by itsPronay**
Optional document translation

fixes issue : https://github.com/kubernetes/website/issues/44993
Related PR : https://github.com/kubernetes/website/pull/45009
Tracking Issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

File: content/bn/includes/partner-style.css
Comment (sftim) [inline content/bn/includes/partner-style.css:1]: We can copy this file verbatim from the English original.

**Comment by asem-hamid** [comment]
@itsPronay  we don't have localize this file . Best you can do copy paste original file .

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**